manifests/init.pp

```diff
@@ -523,6 +523,8 @@
       'dogstatsd_port' => $dogstatsd_port,
       'dogstatsd_socket' => $dogstatsd_socket,
       'dogstatsd_non_local_traffic' => $non_local_traffic,
+      'statsd_forward_host' => $statsd_forward_host,
+      'statsd_forward_port' => $statsd_forward_port,
```

Comment:
These parameters can actually be included in the `$extra_config` hash parameter available for additional options.

Reply:
Yeah, and that's what we're doing now, but then isn't it misleading to have the [corresponding parameters for the actual class](https://github.com/DataDog/puppet-datadog-agent/blob/master/manifests/init.pp#L49-L54) be ignored with agent6? If the intention is to provide those parameters via `extra_config` then should the class parameters then be removed?

Reply:
Let's honor this, the `deep_merge()` should probably be seamless. 

It's tough keeping an interface that was defined for agent5 with the new `datadog.yaml` approach in agent6. But this is a 1-to-1 mapping so no reason not to merge. Sorry we kept it in the backburner for so long.

Thanks.